README.rst

```diff
@@ -1,5 +1,9 @@
 	.. -*- mode: rst -*-
 
+.. image:: https://img.shields.io/pypi/v/nilearn.svg
+    :target: https://pypi.org/project/nilearn/
+    :alt: Pypi Package
+
```

Comment:
```suggestion
.. image:: https://img.shields.io/pypi/v/nilearn.svg
    :target: https://pypi.org/project/nilearn/
    :alt: Pypi Package

.. image:: https://img.shields.io/pypi/pyversions/nilearn.svg
    :target: https://pypi.org/project/nilearn/
    :alt: PyPI - Python Version
```

This is just a thought- but I think it's also useful to have a badge indicating which versions of Python for which the package is supported.

Reply:
Great! thanks! :+1: